mis_builder/static/src/components/mis_report_widget.xml

```diff
@@ -14,6 +14,11 @@
                         <div class="oe_mis_builder_cp_right_bottom">
                             <div class="oe_mis_builder_filter_buttons">
                                 <FilterMenu t-if="showSearchBar" />
+                                <DatePicker
+                                    date="state.pivot_date"
+                                    onDateTimeChanged="onDateTimeChanged.bind(this)"
+                                    t-if="showPivotDate"
+                                />
```

Comment:
```suggestion
                                <DatePicker
                                    date="state.pivot_date"
                                    placeholder="'Base date...'"
                                    onDateTimeChanged="onDateTimeChanged.bind(this)"
                                    t-if="showPivotDate"
                                />
```

Reply:
Add placeholder.
This is the last tweak, I promise :)

Reply:
Don't worry :smile: